container-guide/concepts/containers-docker-registry.xml

```diff
@@ -49,36 +49,33 @@
     <title>Running a &dreg;</title>
     <para>
       The &suseregistry; provides a container image that makes it possible to
-      run a local &dreg; as a container. Before you start a container, create a
-      <filename>config.yml</filename> file with the following example
-      configuration:
-    </para>
-<screen>version: 0.1
-    log:
-      level: info
-    storage:
-      filesystem:
-        rootdirectory: /var/lib/docker-registry
-    http:
-      addr: 0.0.0.0:5000</screen>
-    <para>
-      Also create an empty directory to map the
-      <filename>/var/lib/docker-registry</filename> directory outside the
-      container. This directory is used for storing container images.
+      run a local &dreg; as a container. It stores the pushed images in a
+      container volume corresponding to the directory
+      <filename>/var/lib/docker-registry</filename>. It is recommended to
+      either create a named volume for the registry or to bind mount the
+      directory <filename>/var/lib/docker-registry</filename> onto the host, to
```

Comment:
> onto the host

"into the container"? it reads like the bind mount is operating in an inverted manner

Reply:
I've tweaked the sentence a bit, wdyt?

Reply:
Perfect, thanks!